test/integration/features/basic.feature

```diff
@@ -44,6 +44,15 @@ Feature: Basic
     minishift
     """
 
+  Scenario: Persistent volumes should be listed when minishift start
+    When executing "oc get pv --as system:admin -o=name"
+    Then stderr should be empty
+     And exitcode should equal 0
+     And stdout should contain
+     """
+     persistentvolume/pv0001
+     """
```

Comment:
> for this moment because app deployment still issue for use due to that waiting time.

I think we said we have a solution for that now. _rollout watch_. So writing tests using app deployments should not be an issue more. Biggest "issue" is most likely added test execution time.

However, I don't think you need to deploy an app to make a persistence volume claim. You can claim is directly via the API.